js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -138,11 +149,79 @@ const Dashboard = () => {
           data-testid="dashboard-tabs"
           defaultLocation={toDashboard({
             realm,
-            tab: "info",
+            tab: "welcome",
           })}
           isBox
           mountOnEnter
         >
+          <Tab
+            id="welcome"
+            data-testid="welcomeTab"
+            title={<TabTitleText>{t("welcomeTabTitle")}</TabTitleText>}
+            {...welcomeTab}
+          >
+            <PageSection variant="light">
+              <div className="pf-l-grid pf-u-ml-lg">
+                <div className="pf-l-grid__item pf-m-12-col">
+                  <Title
+                    className="pf-u-font-weight-bold"
+                    headingLevel="h2"
+                    size="3xl"
+                  >
+                    {t("welcome")} {realmDisplayInfo}
+                  </Title>
+                </div>
+                <div className="pf-l-grid__item pf-m-12-col pf-u-mt-md">
+                  <Text component={TextVariants.h3}>
+                    {t("welcomeDescriptionA")}
+                  </Text>
+                  <Text className="pf-u-mt-xs" component={TextVariants.h3}>
+                    {t("welcomeDescriptionB")}
+                  </Text>
+                </div>
+                <div className="pf-l-grid__item pf-m-10-col pf-u-mt-md">
+                  <Button
+                    className="pf-u-px-lg pf-u-py-sm"
+                    component="a"
+                    href="https://www.keycloak.org/documentation"
+                    target="_blank"
+                    variant="primary"
+                  >
+                    {t("viewDocumentation")}
+                  </Button>
+                </div>
+                <div className="pf-u-mt-sm">
```

Comment:
Rather than using utility classes to space out these buttons, perhaps the [action list](https://www.patternfly.org/components/action-list) component would be more appropriate?

Reply:
That's nice @jonkoops - thanks! I was looking for this but couldn't find it.